mmv1/products/networkservices/EndpointPolicy.yaml

```diff
@@ -0,0 +1,152 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'EndpointPolicy'
+base_url: 'projects/{{project}}/locations/global/endpointPolicies'
+create_url: 'projects/{{project}}/locations/global/endpointPolicies?endpointPolicyId={{name}}'
+self_link: 'projects/{{project}}/locations/global/endpointPolicies/{{name}}'
+min_version: beta
+update_verb: :PATCH
+update_mask: true
+description: |
+  EndpointPolicy is a resource that helps apply desired configuration on the endpoints that match specific criteria.
+references: !ruby/object:Api::Resource::ReferenceLinks
+  api: 'https://cloud.google.com/traffic-director/docs/reference/network-services/rest/v1beta1/projects.locations.endpointPolicies'
```

Comment:
note: Not a blocker here, but the cloud.google.com sidebar is broken for these pages.

Reply:
Ack, I will reach out to someone from the docs team